### PR TITLE
Feat/dashboard

### LINKED ISSUE #14: Stworzyć widok dashboarda
Dashboard powinien być zrobiony w stylu dashboarda planetscaleowego/vercelowego.

### PR DESCRIPTION
**Pull request by kalisowski**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,9 +61,12 @@
     "prettier": "^3.0.0",
     "prettier-plugin-tailwindcss": "^0.5.4",
     "tailwindcss": "^3.3.3",
-    "typescript": "^5.1.6"
+    "typescript": "^5.1.6",
+    "bun-types": "latest"
```
Comment (hubcio2115) [inline package.json]: Why did you add this?

File: src/components/authButtons.tsx [hunk 2]
```diff
@@ -4,5 +4,9 @@ import { signOut } from "next-auth/react";
 import { type PropsWithChildren } from "react";
 
 export function SignOutButton({ children }: PropsWithChildren) {
-  return <span onClick={() => signOut()}>{children}</span>;
+  return (
+    <span onClick={() => signOut({ redirect: true, callbackUrl: "/" })}>
```
Comment (hubcio2115) [inline src/components/authButtons.tsx:8]: I think redirect only works, when we have a Credentials provider in auth.js. That is a minor issue and can be ommited.

File: src/hooks/useWindowDimensions.ts
Comment (hubcio2115) [inline src/hooks/useWindowDimensions.ts:1]: What's this for?

**Review by hubcio2115 — CHANGES_REQUESTED** [review]

File: src/hooks/useWindowDimensions.ts
Comment (kalisowski) [reply src/hooks/useWindowDimensions.ts:1]: Max's hook that should be deleted. I forgot about it.

File: src/components/authButtons.tsx (hunk 2 above)
Comment (kalisowski) [reply src/components/authButtons.tsx:8]: Will me omitted

File: package.json (hunk 1 above)
Comment (kalisowski) [reply package.json]: propably added itself when adding some deps. Unused and will be deleted.

File: src/hooks/useWindowDimensions.ts
Comment (hubcio2115) [inline src/hooks/useWindowDimensions.ts:1]: So this stays?

File: src/components/dashboard/dashnav.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  const pathname = usePathname();
+  const session = useSession();
+
+  if (session.status === "unauthenticated") {
+    redirect("/");
+  }
+
+  return (
+    <div>
+      <div className="p- flex items-center justify-between bg-slate-100 p-2">
```
Comment (hubcio2115) [inline src/components/dashboard/dashnav.tsx]: There's an misspell in this class 'p-'

**Review by hubcio2115 — CHANGES_REQUESTED** [review]
I'm gonna make a quick PR to this to fix next/image

File: src/hooks/useWindowDimensions.ts
Comment (kalisowski) [reply src/hooks/useWindowDimensions.ts:1]: removed

File: src/components/dashboard/dashnav.tsx (hunk 3 above)
Comment (kalisowski) [reply src/components/dashboard/dashnav.tsx]: fixed

**Review by hubcio2115 — APPROVED** [review]
LGTM!
